### PR TITLE
NVyrsta

### PR DESCRIPTION
**Pull request by NVyrsta**
[DEMO LINK](https://NVyrsta.github.io/react_people-table-advanced/)

### CODE REVIEW

File: src/components/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  >
+    <div className="container">
+      <div className="navbar-brand">
+        <NavLink
+          className={({ isActive }) => classNames('navbar-item',
+            { 'has-background-grey-lighter': isActive })}
+          to="/"
+        >
+          Home
+        </NavLink>
```
Comment (Bogdan-Kutasevich) [inline src/components/NavBar/NavBar.tsx:20]: this code is repeat, maybe better create new component with all logic inside?

File: src/pages/PeoplePage/PeoplePage.tsx [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    }
+  };
+
+  useEffect(() => {
+    loadPeopleFromServer();
+  }, []);
+
+  const filteredPeople = useMemo(() => {
+    return filterPeople(filterParameters);
+  }, [people, searchParams]);
```
Comment (Bogdan-Kutasevich) [inline src/pages/PeoplePage/PeoplePage.tsx:52]: good idea to create const with filter params)

**Review by Bogdan-Kutasevich — APPROVED** [review]
Nice work! some comments below

File: src/components/PeopleTable/PeopleTable.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+interface Props {
+  people: Person[];
+  selectedPerson: string;
+}
+
+export const PeopleTable: React.FC<Props> = React.memo(({
+  people,
+  selectedPerson,
+}) => {
+  // const sortingFields = ['Name', 'Sex', 'Born', 'Died'];
```
Comment (ArturErmolenko) [inline src/components/PeopleTable/PeopleTable.tsx:16]: delete commented

**Review by ArturErmolenko — APPROVED** [review]
Great job 🚀

**Pull request closed without merging**